kedro/pipeline/node.py

```diff
@@ -677,7 +677,7 @@ def _dict_inputs_to_list(func: Callable[[Any], Any], inputs: Dict[str, str]):
     """Convert a dict representation of the node inputs to a list , ensuring
     the appropriate order for binding them to the node's function.
     """
-    sig = inspect.signature(func).bind(**inputs)
+    sig = inspect.signature(func, follow_wrapped=False).bind(**inputs)
```

Comment:
Can we add a test for this? 🙂

Reply:
Added a test similar to how it was done in #87. It fails if you remove the above change.

Would you prefer a test in a section other than `TestNames`, or is this sufficient?

Reply:
That's alright, thank you!